vnext/Microsoft.ReactNative/Fabric/Composition/ComponentViewRegistry.cpp

```diff
@@ -59,9 +57,6 @@ ComponentViewDescriptor const &ComponentViewRegistry::dequeueComponentViewWithCo
   } else if (componentHandle == facebook::react::ImageShadowNode::Handle()) {
     view = winrt::Microsoft::ReactNative::Composition::implementation::ImageComponentView::Create(
         compContext, tag, m_context);
-  } else if (componentHandle == facebook::react::WindowsModalHostViewShadowNode::Handle()) {
```

Comment:
Quick question: Why is it okay to delete this? Is Modal being registered somewhere that I might have overlooked? 🤔